packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -20,6 +20,9 @@ import Template from 'templates/sign_up_password.mustache';
 
 const t = msg => msg;
 
+const PASSWORD_INPUT_SELECTOR = '#vpassword';
+const DELAY_BEFORE_PASSWORD_CHECK_MS = 2000;
```

Comment:
@ryanfeeley 2 seconds seem reasonable to me while manually testing. Any objections?